metadata/src/test/java/org/apache/kafka/image/loader/MetadataLoaderTest.java

```diff
@@ -249,17 +249,17 @@ public void testPublisherCannotBeInstalledMoreThanOnce(
                 setFaultHandler(faultHandler).
                 setHighWaterMarkAccessor(() -> OptionalLong.of(0L)).
                 build()) {
-            loader.installPublishers(asList(publisher)).get();
+            loader.installPublishers(Collections.singletonList(publisher)).get();
```

Comment:
I wonder if we should import static `singletonList()` in this file because `Collections.singletonList()` is adding a lot of noise compared to the much shorter `asList()`

Reply:
That makes sense. Updating the same!